### PR TITLE
Added textContent setter for ParentNode element.

### PR DESCRIPTION
**Pull request by Indomian**
- Also added special implementation for Document object.

Signed-off-by: Egor Bolgov <e.bolgov@semrush.com>

### CODE REVIEW

File: tests/src/Document.js [hunk 1]
```diff
@@ -85,3 +85,9 @@ test('process query selector all', () => {
     assert.equal(elements[0].textContent, 'Text');
     assert.equal(elements[1].getAttribute('type'), 'text');
 });
+
+test('process textContent', () => {
+    let document = new Document();
+    document.body.innerHTML = '<div><span class="first">Text</span></div><i>Skip me</i><input type="text"/>';
+    assert.equal(document.textContent, null);
```
Comment (christophehurpeau) [inline tests/src/Document.js]: assert.isNull

Comment (Indomian) [inline tests/src/Document.js]: Fixed

**Comment by Indomian** [comment]
Rebased over current master. Fixed code style problems.

**Comment by Indomian** [comment]
Thank you!

**Pull request merged**